test/core/codec/cbor/cbor_test.cpp

```diff
@@ -73,6 +73,22 @@ TEST(Cbor, Null) {
   EXPECT_FALSE(CborDecodeStream("01"_unhex).isNull());
 }
 
+/** Optional CBOR encoding and decoding */
```

Comment:
And here given-when-then, too, please. It just helps to write more meaningful tests, for edge cases for example.